specs/grafana/SOURCES/grafana-server.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Grafana instance
+Documentation=http://docs.grafana.org
+Wants=network-online.target
+After=network-online.target
+After=postgresql.service mariadb.service mysqld.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/grafana-server
+User=grafana
+Group=grafana
+Type=notify
+Restart=always
```

Comment:
Please add `StartLimitIntervalSec` and `StartLimitBurst`.

Reply:
Fixed (4ab1291589772c484293d167c0937ffef72612fc)